docs/en/stock-management/movement.exit.md

```diff
@@ -2,29 +2,221 @@
 
 # Stock Exit
 
-Any outflows of stock from a depot is a stock exit in BHIMA.  The available options are:
+Any outflows of stock from a depot is a stock exit in BHIMA. The exit of stock
+concerns a stock pharmacy as well as a dispensing pharmacies.
```

Comment:
This is somewhat awkward phrasing.

Let's say "any movement of stock out of a depot is a stock exit in BHIMA.  Stock exits can happen from any depot, including the stock and dispensing pharmacies."

Reply:
Fixed